.github/workflows/ci-windows.yml

```diff
@@ -58,6 +58,12 @@ jobs:
         run: npm install -g bpmn-js-cmd
       - name: Install Bytefield-SVG
         run: npm install -g bytefield-svg
+      - name: Install Dpic
+        run: |
+          mkdir dpic
+          Invoke-WebRequest -Uri "https://ece.uwaterloo.ca/~aplevich/dpic/Windows/dpic.exe" -OutFile dpic.exe
+          mv dpic.exe dpic
```

Comment:
Possibly you can avoid `mv` by passing `-OutFile dpic\dpic.exe` to `Invoke-WebRequest` on the previous line?

Reply:
@slonopotamus sure